aws/kubectl-manifest/main.tf

```diff
@@ -0,0 +1,3 @@
+resource "kubectl_manifest" "secret_retool_config" {
```

Comment:
it might be worth while to parse this as yaml using a data source, to make sure we have valid yaml.

IE: https://developer.hashicorp.com/terraform/language/functions/yamldecode and then pass the result in.